explore/src/main/scala/explore/ExploreLayout.scala

```diff
@@ -64,6 +66,12 @@ object ExploreLayout:
       .withHooks[Props]
       .useContext(HelpContext.ctx)
       .useContext(AppContext.ctx)
+      .useEffectWithDepsBy((p, _, _) => p.view.zoom(RootModel.programReference).get)((_, _, _) =>
+        // Set the title of the page to the program reference
+        // scalajs-react suggest to use setTitle for this but we'd need to put the
+        // reference in the url for that
+        _.map(r => Callback(document.title = s"Explore - ${r.label}")).orEmpty
```

Comment:
Is it possible to get an update that removes the program id? If so this would keep the old one in the title

Reply:
This would also be a problem when switching from a program with a program reference to a program without a reference, wouldn't it?